Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -562,16 +684,21 @@ public class ProgramBuilder {
 
         generator.run(in: self, with: inputs)
     }
-    
+
+    /// Executes a code template.
+    func run(_ template: CodeTemplate) {
```

Comment:
I think we can avoid this additional indirection and just call `template.run(in: b)` directly in the HybridFuzzer.

Reply:
And maybe `run` isn't the ideal name, how about `template.generate(in: b)` or something like that?